src/woff2_dec.cc

```diff
@@ -1061,31 +1317,29 @@ bool ConvertWOFF2ToTTF(uint8_t* result, size_t result_length,
           transform_length) > result_length)) {
         return FONT_COMPRESSION_FAILURE();
       }
+      transform_buf = src_by_dest.at(table->dst_offset);
```

Comment:
@rsheeter ...and I wonder whether you'd be happy to use src_by_dest[table->dst_offset] here. This is essentially the same except that it will add an item in table->dst_offset if it is absent instead of raising an exception. But it does not require C++11, so that will help for Mozilla local build (see https://bugzilla.mozilla.org/show_bug.cgi?id=1227058)